flow-plugins/flow-maven-plugin/src/test/java/com/vaadin/flow/plugin/maven/BuildFrontendMojoTest.java

```diff
@@ -142,12 +142,6 @@ public void setup() throws Exception {
                 "jar-resources-source/META-INF/frontend");
         jarResourcesSource.mkdirs();
 
```

Comment:
Should we put this back as the original change was reverted as I see?:
```suggestion

        File statsfile = new File(resourceOutputDirectory,
                Constants.VAADIN_CONFIGURATION + "/stats.json");

        statsfile.getParentFile().mkdirs();
        FileUtils.fileWrite(statsfile, "UTF-8", "{}");
```

Reply:
It should not be needed now that `{}` is used when stats.json is not found

Reply:
yes, correct, I was on autopilot, morning coffee needs before reviews!